internal/db/advisory_locks.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2025 Canonical.
+
+package db
+
+import (
+	"context"
+	"errors"
+	"fmt"
+)
+
+// LockID represents a unique identifier for an advisory lock in the database.
+type LockID int64
+
+const (
+	// ControllerBootstrapLock is the advisory lock ID used for controller bootstrap operations.
```

Comment:
perhaps a godoc that we should NOT EVER change this ID..

Reply:
Fs I just merged and didn't see these comments. I'll add this in this one https://github.com/canonical/jimm/pull/1627/ if that's ok?